src/constants.ts

```diff
@@ -33,6 +34,8 @@ export enum ProjectLanguage {
     Custom = 'Custom'
 }
 
+export const previewPythonModel: number = 2;
```

Comment:
Could you explain what these numbers mean? I'm not really sure what the significances of the numbers are for the models, but you set a minimum of 1 for the setting.

Reply:
The number itself is arbitrary and just corresponds to a specific iteration of the language model.  `1` is the (current) model (and assumed if omitted).  `2` is the "new" model (with binding attributes).  If there are any significant changes made to the model in the future (such that tooling would need to distinguish between it and previous iterations), that model would be assigned an incremented number.